lua/lvim/lsp/templates.lua

```diff
@@ -42,6 +42,10 @@ end
 ---The files are generated to a runtimepath: "$LUNARVIM_RUNTIME_DIR/site/after/ftplugin/template.lua"
 ---@param servers_names table list of servers to be enabled. Will add all by default
 function M.generate_templates(servers_names)
+  if package.loaded["lspconfig"] then
+    pcall(vim.cmd, "LspStop")
```

Comment:
I'm not sure if pcall with a vim.cmd will work or not.

If you ever want to look under the hood and see what the lua is for a command you can just use `:command LspStop` etc. to list what it is.
For example LspStop is: `require'lspconfig'["_root"].commands["LspStop"][1](<f-args>)` where `<f-args>` are the arguments passed to LspStop in the vim command. For this case, there are none so: `require'lspconfig'["_root"].commands["LspStop"][1]()`

Reply:
In this case I know what `LspStop` does, it's supposed to stop all active clients, which is actually easier to write directly compared to `LspRestart`.

Since I do recall seeing it, I did a quick search and out of all the plugins I use, these are all occurrences

![image](https://user-images.githubusercontent.com/59826753/148736252-c52e2876-8514-4a20-acbb-a31422d34032.png)

Maybe it doesn't actually work 😅

Reply:
Ah, to explain, I meant using pcall for handling an error from the vimscript runtime might not work. I'm not certain about this.

Reply:
Yeah, I got what you meant. I've switched it to use the lua API directly instead.